scripts/helpers

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+# Validates dev_tags for DockerHub only pushes to "hashicorppreview" org
+dev_tags_validation() {
+  if [ -n "${1}" ]; then
+    for dt in ${1}; do
+      # strip docker.io if exists
+      dt_stripped=${dt//docker.io\//}
+
+      # if there are two parts we presume it's the public docker registry, ex: hashicorppreview/consul. Anything else is a registry we don't check currently.
+      parts=$(echo "$dt_stripped" | awk -F'/' '{print NF}')
+      if [ "$parts" -eq 2 ] && [[ "$dt_stripped" != hashicorppreview/* ]]; then
+        echo "Only valid DockerHub organization is 'hashicorppreview' (Got: $dt)"
```

Comment:
I might be tempted to make this error message even more specific, since we are also rejecting any non-DockerHub tags. E.g. `"dev_tags must begin with 'hashicorppreview/' or 'docker.io/hashicorppreview/'"` ?

Reply:
We should also put this message on stderr, since many scripts in this project use stdout to pass data around. (In this specific case, I don't think it will break anything, but best to keep stdout clean to be consistent, and avoid tripping us up if we start wanting stdout to be used for something in the future.) I put a `log` function at the top of most scripts to enable easy writing messages to stderr; `log() { echo "$*" 1>&2; }`